bitnami/mariadb/templates/update-password/job.yaml

```diff
@@ -0,0 +1,241 @@
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "mariadb.update-job.previousSecretName" . }}
+                  key: mariadb-password
+            - name: MARIADB_NEW_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "mariadb.update-job.newSecretName" . }}
+                  key: mariadb-password
+            {{- end }}
+            {{- end }}
+            {{- if eq .Values.architecture "replication" }}
+            - name: MARIADB_REPLICATION_USER
+              value: {{ .Values.auth.replicationUser | quote }}
+            {{- if not .Values.auth.usePasswordFiles }}
+            - name: MARIADB_PREVIOUS_REPLICATION_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "mariadb.update-job.previousSecretName" . }}
+                  key: mariadb-replication-password
+            - name: MARIADB_NEW_REPLICATION_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "mariadb.update-job.newSecretName" . }}
+                  key: mariadb-replication-password
+            {{- end }}
+            {{- end }}
+            {{- if .Values.passwordUpdateJob.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.passwordUpdateJob.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          {{- if or .Values.passwordUpdateJob.extraEnvVarsCM .Values.passwordUpdateJob.extraEnvVarsSecret }}
+          envFrom:
+            {{- if .Values.passwordUpdateJob.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ .Values.passwordUpdateJob.extraEnvVarsCM }}
+            {{- end }}
+            {{- if .Values.passwordUpdateJob.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ .Values.passwordUpdateJob.extraEnvVarsSecret }}
+            {{- end }}
+          {{- end }}
+          {{- if .Values.passwordUpdateJob.containerSecurityContext.enabled }}
+          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.passwordUpdateJob.containerSecurityContext "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.passwordUpdateJob.customLivenessProbe }}
+          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.passwordUpdateJob.customLivenessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.passwordUpdateJob.customReadinessProbe }}
+          readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.passwordUpdateJob.customReadinessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.passwordUpdateJob.customStartupProbe }}
+          startupProbe: {{- include "common.tplvalues.render" (dict "value" .Values.passwordUpdateJob.customStartupProbe "context" $) | nindent 12 }}
+          {{- end }}
```

Comment:
Are these required in a job?

Reply:
If they tried to follow DISA/CIS guidelines, they might want to set them. In the end we decided to not add them in jobs because in most cases it did not make sense.

Reply:
> In the end we decided to not add them in jobs because in most cases it did not make sense.

I'm confused.. Default value for `passwordUpdateJob.livenessProbe.enabled` and `passwordUpdateJob.readinessProbe.enabled` is `true`.

Reply:
Leftover from copy-paste, removed